tests/end_to_end/features/autovariants.feature

```diff
@@ -0,0 +1,47 @@
+Feature: AutoVariants
+    Scenario: Same auto variant for same transformation 
+        Given Featureform is installed
+        When I turn on autovariants
```

Comment:
i was wrong about this, simba pointed out a good observation, since things are run in parallel this won't have the affect we want